ui/packages/ace/lib/ls-packages.js

```diff
@@ -7,9 +7,8 @@ module.exports = (dirs) => {
   return dirs
     .filter((dir) => !dir.match('target'))
     .map((d) => path.resolve(d))
-    .map((d) => glob.sync(d))
+    .map((d) => glob.sync(d + '/package.json'))
```

Comment:
❓ So basically because `package.json` files are version controlled, any directory in `packages` that doesn't have a `package.json` in it won't match this glob and won't be used by ace?

Reply:
HA! Empty directories - take that! 
https://media.giphy.com/media/AhM9s5lvgw78k/giphy.gif